host/libvirt_lxc_backend.go

```diff
@@ -76,6 +76,7 @@ type LibvirtLXCBackend struct {
 	state     *State
 	pinkerton *pinkerton.Context
 
+	ifaceMTU   int
```

Comment:
I don't feel strongly about removing it, just looked like an odd diff introducing a field which isn't used